config/sample_hanatpcc_config.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<parameters>
+	
+    <!-- Connection details -->
+    <dbtype>saphana</dbtype>
+    <driver>com.sap.db.jdbc.Driver</driver>
+    <DBUrl>jdbc:sap://ottertune-dev.pdl.cmu.edu:39015/tpcc</DBUrl>
+    <username>SYSTEM</username>
+    <password>1234Aa*-</password>

```

Comment:
Is this their password or yours? I would make this generic. See other examples.

Reply:
change to user and password